### DISCUSSION TITLE
[Feature request] Allow Shape game object to be a mask

### DISCUSSION THREAD

**Discussion by rexrainbow** [photonstorm/phaser#5900] [Feature Requests]
`createGeometryMask` method only accepts Graphics game object to be a mask, but Shape game objects also could be a mask too,  [in my test](https://codepen.io/rexrainbow/pen/oNwjZQm). After changing `type` property to `'Graphics'` for [this line](https://github.com/photonstorm/phaser/blob/master/src/gameobjects/components/Mask.js#L132).

**Comment by photonstorm** [comment]
Thank you for submitting this feature request. We have implemented this and the feature has been pushed to the `master` branch. It will be part of the next release. If you get time to build and test it for yourself we would appreciate that.

**Comment by erwingame** [comment]
Good advice.
